examples/switch.rb

```diff
@@ -0,0 +1,31 @@
+# (c) Copyright 2016 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+my_client = {
+  url: '',
+  user: '',
+  password: ''
+}
+
+# Example: No action is executed.
+# In a resource that has only one action and no action is specified in the block
+# CHEF executes this one. To prevent CHEF from removing a switch as the standard action we created
```

Comment:
Change CHEF to Chef. Also it could be more concise like 
```Ruby
# Since for most of the resources the default action is :add, the action :none was created and was set to default.
# It will prevent the default action to be :remove and cause accidental removes
```